polynote-spark/src/main/scala/polynote/kernel/SparkPolyKernel.scala

```diff
@@ -31,15 +31,15 @@ import scala.tools.nsc.interactive.Global
 // TODO: Should the spark init stuff go into the Spark Scala kernel? That way PolyKernel could be the only Kernel.
 class SparkPolyKernel(
   getNotebook: () => IO[Notebook],
-  globalInfo: GlobalInfo,
+  kernelContext: KernelContext,
   dependencies: Map[String, List[(String, File)]],
   statusUpdates: Publish[IO, KernelStatusUpdate],
   outputPath: Path,
-  subKernels: Map[String, LanguageKernel.Factory[IO]] = Map.empty,
+  subKernels: Map[String, LanguageInterpreter.Factory[IO]] = Map.empty,
   parentClassLoader: ClassLoader
-) extends PolyKernel(getNotebook, globalInfo, new PlainDirectory(new Directory(outputPath.toFile)), dependencies, statusUpdates, subKernels) {
+) extends PolyKernel(getNotebook, kernelContext, new PlainDirectory(new Directory(outputPath.toFile)), dependencies, statusUpdates, subKernels) {
 
-  val global = globalInfo.global
+  val global = kernelContext.global
```

Comment:
Should this be an import instead, per your comment? (maybe we should call it `globalImportOnlyNoVal` 😜 )

Reply:
Yes, absolutely right. Good catch!

Reply:
Fixed